tests/API1/testtupleparam.py

```diff
@@ -139,13 +139,14 @@ class A(param.Parameterized):
         class B(A):
             p = param.Tuple()
 
-        # B inherits the default (0, 0) and its computed length
-        assert B.p == (0, 0)
-        assert B.param.p.length == 2
+        assert B.p == (0, 1, 2)
+        assert B.param.p.length == 3
```

Comment:
This is testing that `length` is computed based on the default value in the sub class rather than from the default in the super class. I got this wrong at some point and remember that `length` was 2 while `B.p` was `(0, 1, 2)`.